slides/02_Setup.html

```diff
@@ -42,15 +42,32 @@ <h2>How React can be used</h2>
               server-side rendering (SSR) and static site generation (SSG).
             </p>
             <p class="fragment">
-              For our demo app, we will start with Vite and compare some aspects
-              later on with a framework supporting SSR.
+              For our demo app, we will start with Vite (chapters 2-9) and
+              compare some aspects later (chapter 10) on with a framework
+              supporting SSR.
             </p>
             <p class="fragment tip">
               For all those that know
               <em>Create React App</em>, please notice that this project is
               discontinued since April 2022.
             </p>
           </section>
+          <section>
+            <h2>A word about client side rendering</h2>
+            <p class="fragment">
+              When reading up on
+              <a href="https://react.dev/">react.dev</a>, you will notice that
+              pure client side rendering — like with Vite — is not mentioned at
+              all.
+            </p>
+            <p class="fragment">
+              We believe that pure client side rendering still has it's benefits
+              for some usecases (i.e. company internal app) and is a valid pick
```

Comment:
Nitpicks:
- _use cases_ (separated)
- I think you want to use _e.g._ here instead of _i.e._ (see https://www.merriam-webster.com/grammar/ie-vs-eg-abbreviation-meaning-usage-difference)
- _due to **its** reduced **overall** complexity_